scripts/start.js

```diff
@@ -58,13 +59,22 @@ const devServer = new WebpackDevServer(compiler, {
   quiet: true
 });
 
-// Launch WebpackDevServer.
-devServer.listen(port, function (err) {
+detect(port, function (err, _port) {
+
   if (err) {
-    return console.log(err);
+    console.log(err);
   }
-});
 
-openBrowser('http://localhost:' + port + '/');
+  var usePort = (port === _port) ? port : _port;
```

Comment:
I agree, there's some more work to be done, I think having a prompt would be a reasonable addition.